site/src/components/pages/hub/BlockTabsModal.tsx

```diff
@@ -26,7 +26,15 @@ export const BlockTabsModal: VoidFunctionComponent<BlockTabsModalProps> = ({
   setText,
 }) => {
   return (
-    <Modal open={open} onClose={() => setOpen((oldValue) => !oldValue)}>
+    <Modal
+      open={open}
+      onClose={() => setOpen((oldValue) => !oldValue)}
+      sx={{
+        "& .MuiBackdrop-root": {
+          backgroundColor: "rgba(77, 92, 108, 0.6)",
```

Comment:
Instead of using `rgba()`, you could use one of the theme palette colours and set the backdrop's opacity to `0.6`, or whatever else you'd like.

Reply:
Fixed here acef6d4. Couldn't use opacity since mui modal uses that internally for transitions. Realized mui exports an alpha method which can be used instead. So we can do

```
{ backgroundColor: ({ palette }) => alpha(palette.gray[70], 0.6) }
```